packages/gatsby-theme-newrelic/gatsby/type-defs.js

```diff
@@ -1,11 +1,56 @@
-const fs = require('fs');
-const path = require('path');
+const SCHEMA_CUSTOMIZATION_TYPES = `
+  type SiteLayout @dontInfer {
+    contentPadding: String
+    maxWidth: String
+    mobileBreakpoint: String
+  }
+
+  type MdxFrontmatter @infer {
+    startDate: Date @dateformat(formatString: "YYYY-MM-DD")
+    endDate: Date @dateformat(formatString: "YYYY-MM-DD")
+  }
+
+  type SiteSiteMetadata {
+    repository: String
+    branch: String!
+    contributingUrl: String
+    title: String
+    titleTemplate: String
+  }
+
+  type Locale implements Node @dontInfer {
+    name: String!
+    localName: String!
+    locale: String!
+    hrefLang: String!
+    isDefault: Boolean!
+  }
+
+  type RelatedResource implements Node {
+    id: ID!
+    title: String!
+    url: String!
+  }
+
+  type NewRelicThemeConfig implements Node {
+    env: String!
+    relatedResources: NewRelicThemeRelatedResourceConfig!
+    tessen: NewRelicThemeTessenConfig
+  }
+
+  type NewRelicThemeRelatedResourceConfig {
+    labels: [RelatedResourceLabel!]!
+  }
+
+  type RelatedResourceLabel {
+    baseUrl: String!
+    label: String!
+  }
 
-const SCHEMA_CUSTOMIZATION_TYPES = fs.readFileSync(
-  path.resolve(__dirname, './type-defs.graphql'),
-  {
-    encoding: 'utf-8',
+  type NewRelicThemeTessenConfig {
+    product: String
+    subproduct: String
   }
-);
+`;
 
```

Comment:
Build with failing with `fs`, we came up with this as a possible solution. Open to different ideas!